src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/Modal/feedback-filter-modal.component.html

```diff
@@ -0,0 +1,74 @@
+<form [formGroup]="filterForm" (ngSubmit)="applyFilter()">
+    <div class="modal-header">
+        <h5 class="modal-title" jhiTranslate="artemisApp.programmingExercise.configureGrading.feedbackAnalysis.filterModal.modalTitle"></h5>
+        <button type="button" class="btn-close" (click)="closeModal()"></button>
+    </div>
+    <div class="modal-body">
+        <div class="form-group">
+            <label jhiTranslate="artemisApp.programmingExercise.configureGrading.feedbackAnalysis.filterModal.task"></label>
+            <div class="row">
+                @for (task of generateTaskArray(); track task) {
```

Comment:
Would it be possible to use a `computed` signal instead of a function call here?

Something like

```ts
readonly taskArray = computed(() => Array.from({ length: this.totalAmountOfTasks() }, (_, i) => i + 1));
```